### PR TITLE
Feature/footer

### PR DESCRIPTION
**Pull request by GiovaneCardoso**
## What's the purpose of this pull request?

Create a footer component

## How to test it?

By preview generated with this commit

### CODE REVIEW

File: public/icons.svg [hunk 1]
```diff
@@ -1,53 +1,380 @@
...
-<symbol id="ArrowLeft" xmlns="http://www.w3.org/2000/svg" fill="currentColor" viewBox="0 0 256 256"><rect width="256" height="256" fill="none"></rect><line x1="216" y1="128" x2="40" y2="128" fill="none" stroke="currentColor" stroke-linecap="round" stroke-linejoin="round"></line><polyline points="112 56 40 128 112 200" fill="none" stroke="currentColor" stroke-linecap="round" stroke-linejoin="round"></polyline></symbol>
-<symbol id="ArrowUpRight"  xmlns="http://www.w3.org/2000/svg" width="256" height="256" viewBox="0 0 256 256" fill="none"><path d="M13,0.25H3.25C2.83579,0.25,2.5,0.585786,2.5,1C2.5,1.41421,2.83579,1.75,3.25,1.75H11.1893L0.46967,12.4697C0.176777,12.7626,0.176777,13.2374,0.46967,13.5303C0.762563,13.8232,1.23744,13.8232,1.53033,13.5303L12.25,2.81066V10.75C12.25,11.1642,12.5858,11.5,13,11.5C13.4142,11.5,13.75,11.1642,13.75,10.75V1C13.75,0.793702,13.6667,0.606858,13.5319,0.471263C13.5314,0.470731,13.5309,0.4702,13.5303,0.46967C13.5298,0.46914,13.5293,0.46861,13.5287,0.468082C13.4572,0.396963,13.3749,0.343239,13.2871,0.306909C13.1987,0.270239,13.1017,0.25,13,0.25Z" fill="black"/></symbol>
```
Comment (andrechism) [inline public/icons.svg:12]: Try to maintain this file with one line symbols for better reading, like it was here.

You could use this [xml-minifier](https://www.webtoolkitonline.com/xml-minifier.html) to minify all this content and then replace all the close tags with a line break 😋
image

File: public/icons.svg [hunk 2]
```diff
@@ -1,53 +1,380 @@
...
+        <stop offset="0.069" stop-color="#2DA771" />
+        <stop offset="0.476" stop-color="#15CF74" />
+        <stop offset="0.801" stop-color="#06E775" />
+        <stop offset="1" stop-color="#00F076" />
+      </linearGradient>
+      <clipPath id="clip0_2690_7357">
+        <rect width="118.125" height="35" fill="white" transform="translate(0.125)" />
+      </clipPath>
+    </defs>
+  </svg>
```
Comment (andrechism) [inline public/icons.svg:349]: You should use a symbol tag instead of svg

File: src/components/common/Footer/Badges/index.tsx [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+import { data } from './data'
+import styles from './badges.module.scss'
+
+const Badges = () => {
+  return data ? (
+    <div className={styles.footerBadgeContainer}>
+      <div className={styles.footerBadgeContainerInner}>
+        {data.map((badge, index) => (
+          <div className={styles.badgeItem} key={index}>
```
Comment (andrechism) [inline src/components/common/Footer/Badges/index.tsx:11]: Try to use something else than only an index for a key
I would do something like this:
image


other files with the same issue:
[src/components/common/Footer/NavLinks/LinksList/MobileLink/index.tsx](https://github.com/vtex-sites/casinofr.store/blob/70df95dc3d54aea3eb107651df3b2d28d8991971/src/components/common/Footer/NavLinks/LinksList/MobileLink/index.tsx)
[src/components/common/Footer/NavLinks/LinksList/index.tsx](https://github.com/vtex-sites/casinofr.store/blob/70df95dc3d54aea3eb107651df3b2d28d8991971/src/components/common/Footer/NavLinks/LinksList/index.tsx)


more info: 
https://reactjs.org/docs/lists-and-keys.html#keys
https://robinpokorny.medium.com/index-as-a-key-is-an-anti-pattern-e0349aece318

File: src/components/common/Footer/Social/index.tsx [hunk 4]
```diff
@@ -0,0 +1,198 @@
+import Icon from 'src/components/ui/Icon'
+
+import styles from './social.module.scss'
+
+const GooglePlayIcon = () => (
```
Comment (andrechism) [inline src/components/common/Footer/Social/index.tsx:5]: Create another file for this component o(*￣▽￣*)ブ

File: public/icons.svg (hunk 2 above)
Comment (evander-corebiz) [reply public/icons.svg]: removed

File: src/components/common/Footer/Social/index.tsx (hunk 4 above)
Comment (evander-corebiz) [reply src/components/common/Footer/Social/index.tsx]: done

File: src/components/common/Footer/NavLinks/data.ts [hunk 5]
```diff
@@ -0,0 +1,95 @@
+export const data = [
```
[the remark this answers is not in the document]
Comment (evander-corebiz) [reply src/components/common/Footer/NavLinks/data.ts:1]: done

**Pull request merged**